ci/travis/determine_tests_to_run.py

```diff
@@ -149,25 +149,30 @@ def list_changed_files(commit_range):
         RAY_CI_ONLY_RLLIB_AFFECTED = 1
 
     # Log the modified environment variables visible in console.
-    for output_stream in [sys.stdout, sys.stderr]:
-        _print = partial(print, file=output_stream)
-        _print("export RAY_CI_TUNE_AFFECTED={}".format(RAY_CI_TUNE_AFFECTED))
-        _print("export RAY_CI_ONLY_RLLIB_AFFECTED={}"
+    for output_stream in [sys.stdout]:
```

Comment:
`for output_stream in [sys.stdout, sys.stderr]:` is intentional to help debug which files are affected and the the env var. It's quite important.

Reply:
Yeah but it's [already being printed by the parent script](https://travis-ci.com/github/ray-project/ray/jobs/318792471#L213) when `set -x` is enabled; this part is redundant (floods the screen).

Reply:
oh ok i didn't notice that. in this case can you just change `_print` to normal print and remove the for loop?

Reply:
Okay sure!